app/models/authentication.rb

```diff
@@ -118,15 +119,6 @@ def self.build_credential_options
     end
   end
 
-  def self.class_from_request_data(data)
```

Comment:
@jrafanie yeah this is the only caller - this was a method I created for the API back in March to be replaced by this update